pythonsd/static/js/main.js

```diff
@@ -0,0 +1,5 @@
+// Toggle the navigation menu on small screens
+document.getElementById("navigation-toggle").addEventListener("click", function () {
+    const navMenu = document.querySelector("#navigation-menu");
```

Comment:
```suggestion
    const navMenu = document.getElementById("navigation-menu");
```
getElementById() is faster than querySelector(), [source](https://stackoverflow.com/a/71878632). Obviously the performance difference won't be noticeable, I just like this optimization.

Personally, I would also combine lines 3 and 4 and not bother with assigning `navMenu`. Less readable, though, so I'm not suggesting to change it.

Reply:
Looks good to me. Thanks.